backend/user-service/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Please consider adding scripts like "dev", "build", "test", etc. as they are very helpful in running the server locally.

Reply:
added script for dev